packages/ui/src/button/Button.tsx

```diff
@@ -124,13 +128,10 @@ const Button = ({
       {...rest}
       disabled={disabled}
     >
-      {/* left-icon */}
-      {/* <Text type={textStyles()}>{text?.length > 0 ? text : children}</Text> */}
-      {children}
-      {/* right-icon */}
+      {/* left-icon은 여기로 */}
+      <div>{text?.length > 0 ? text : children}</div>
```

Comment:
```suggestion
      <div>{text ?? children}</div>
```
여기서 text 를 props 로 안줬을때 children을 렌더링하고 싶었던거져? 그럼 요렇게 바꿔도 될거에여